Initial load of Thanksgiving post (#481)
oktadeveloper/okta-blog: _source/_posts/2020-11-25-open-source-thankful.md

---
layout: blog_post
title: "Five Open Source Projects We're Thankful For In 2020"
author: lee-brandt
by: advocate
communities: [devops]

> description: "A list of five open source projects that the Dev Relations team likes."

Reviewer: ```suggestion
description: "A list of five open source projects that the Okta Developer Relations team is thankful for."
```
